sdk/cosmos/microsoft-azure-cosmos/src/test/java/com/azure/data/cosmos/rx/OrderbyDocumentQueryTest.java

```diff
@@ -408,21 +395,26 @@ public void beforeMethod() throws Exception {
 
     @BeforeClass(groups = { "simple" }, timeOut = SETUP_TIMEOUT)
     public void beforeClass() throws Exception {
+        System.out.println("OrderbyDocumentQueryTest.beforeClass");
```

Comment:
We should use logger here and at other places in this class. 
It is available from TestSuiteBase, which gets it from DocumentClientTest

Reply:
Still under debug. I dont need these and I will remove

Reply:
Removed